packages/create-pantheon-decoupled-kit/src/templates/partials/nextjs-shared/nextjsLayoutCSS.hbs

```diff
@@ -21,6 +21,10 @@
 	color: var(--white);
 }
 
-.footerCopy > a[href$='wordpress.com/'] {
+.footerCopy > a[href$={{#if wp}}'wordpress.com/'{{else if drupal}}'drupal.org/'{{/if}}] {
```

Comment:
Using a selector like `:last-child` or `:nth-child()` would allow the hbs conditional to be eliminated here.